Resources/Prototypes/Entities/Objects/Consumable/Food/ingredients.yml

```diff
@@ -463,6 +463,40 @@
         - ReagentId: Nutriment
           Quantity: 5
 
+- type: entity
+  name: tofu
+  parent: FoodBakingBase
+  id: FoodTofu
+  description: Solid white block with a subtle flavor.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - tofu
+  - type: Sprite
+    state: tofu
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 6
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 5
```

Comment:
It has 3 slices, so should contain more nutrition. I'd go 9 Protein, 6 Nutriment, as Tofu is protein-rich.

Reply:
You've got a point